src/main/java/htsjdk/variant/vcf/VCFUtils.java

```diff
@@ -250,6 +254,32 @@ else if (vcfFile.getAbsolutePath().endsWith(IOUtil.COMPRESSED_VCF_FILE_EXTENSION
         return output;
     }
 
+    /**
+     * Parses a String as a Double, being tolerant for lower-case NaN (nan).
```

Comment:
This javadoc doesn't seem sufficient given that this supports INF and NaN. It would be nice to document its behavior more completely, and/or provide a link to part of the spec that this code is implementing.